app/javascript/components/works/Works.jsx

```diff
@@ -21,54 +21,57 @@ class Works extends React.Component {
       filters: {},
       isLoading: true,
       pageCount: 0,
-      workStartIndex: 0,
-      workEndIndex: 0
+      currentPage: 0,
+      filtering: false
     };
   }
 
   componentDidMount = () => {
     const works_route = this.props.userType == "admin"
-      ? APIRoutes.works.index
-      : APIRoutes.works.filtered_artist_hidden
+      ? APIRoutes.works.index(1)
+      : APIRoutes.works.filtered_artist_hidden(1)
     const categories_route = APIRoutes.works.categories;
     Promise.all([
       Requester.get(works_route),
       Requester.get(categories_route)
     ]).then(response => {
       const [works_response, filters_response] = response;
       this.setState({
-        works: works_response,
+        works: works_response.works,
         filters: filters_response,
         isLoading: false,
-        pageCount: Math.ceil(works_response.length / perPage),
-        workStartIndex: 0,
-        workEndIndex: perPage
+        pageCount: Math.ceil(works_response.work_count / perPage)
```

Comment:
perPage should not be hardcoded into the front end! Send it back as part of the pagination response as well so there's only 1 source of truth.